common/helpers/field.js

```diff
@@ -177,14 +179,59 @@ function insertItemConditional({ key, field }) {
   }
 }
 
+async function mapAssessmentQuestion([key, field]) {
+  if (!Object.prototype.hasOwnProperty.call(field, 'items')) {
+    return [key, field]
+  }
+
+  const assessmentQuestions = await referenceDataService.getAssessmentQuestions(
+    key
+  )
+
+  field.items = assessmentQuestions
+    .filter(referenceDataHelpers.filterDisabled())
+    .map(mapAssessmentQuestionToConditionalField)
+    .map(mapAssessmentQuestionToTranslation)
+    .map(mapReferenceDataToOption)
+
+  return [key, field]
+}
+
+function mapAssessmentValidation([key, field], index, obj) {
+  if (!isNull(field.dependent)) {
```

Comment:
Based on the earlier comment I think it would make this easier to read too. It would become "If field is not required" rather than "If dependent property is not null".

Reply:
Agreed - it appears that `validate` can only have the string `'required'` so a check for `field.validate` should suffice here